### PR TITLE
feat: added results page

### PR DESCRIPTION
**Pull request by partha8**
Hey everyone, I have added the results page. Please share your views. Thank you

Preview: https://deploy-preview-4--algern-quiz.netlify.app/components/result-page.html

### CODE REVIEW

**Comment by Dev-AkshatGupta** [comment]
Please use cursor to pointer on the div of selecting the options

**Comment by partha8** [comment]
> Please use cursor to pointer on the div of selecting the options

I have disabled the cursor:pointer on the results page, because I don't want the user to get the idea that they can click the options.

File: components/result-page.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    <main>
+        <h2 class="text-center">Results: 5/10</h2>
+        <section class="question-container">
+            <img class="responsive-image question-img" src="../assets/holding-al-pacino-best-and-worst-film-moments.jpg"
+                alt="al pacino">
+            <article class="mcq-container">
+                <h2 class="m-0">Who is this actor?</h2>
+                <h3 class="m-0">Points: 5</h3>
+                <div class="mcq">
```
Comment (foolhardy21) [inline components/result-page.html]: I think instead of MCQ you can give some other name like choices or options or something else more suitable.

**Review by foolhardy21 — APPROVED** [review]
rest is good to merge.

File: components/result-page.html (hunk 1 above)
Comment (partha8) [reply components/result-page.html]: I will certainly make the required changes, thank you @foolhardy21.

**Review by vrx29 — APPROVED** [review]
Looks good to me. You can go ahead and merge

**Review by Dev-AkshatGupta — APPROVED** [review]
Else everything seems fine to me

**Comment by Dev-AkshatGupta** [comment]
> > Please use cursor to pointer on the div of selecting the options
> 
> I have disabled the cursor:pointer on the results page, because I don't want the user to get the idea that they can click the options.

Why so didn't got your idea behind that??

**Comment by partha8** [comment]
> > > Please use cursor to pointer on the div of selecting the options
> > 
> > 
> > I have disabled the cursor:pointer on the results page, because I don't want the user to get the idea that they can click the options.
> 
> Why so didn't got your idea behind that??

On a result page, when the user goes over their performance, having a cursor:pointer on the options here won't make much sense and it might be misleading right? Whereas, having cursor:pointer on the question page will send a clear indication that they have to choose one option. I hope I am making sense here.

**Pull request merged**